packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -542,19 +550,31 @@ export default class DatePicker extends React.Component {
     }
 
     return (
-      <PopperComponent
-        className={this.props.popperClassName}
-        hidePopper={(!this.state.open || this.props.disabled)}
-        popperModifiers={this.props.popperModifiers}
-        targetComponent={
-          <div className="react-datepicker__input-container">
-            {this.renderDateInput()}
-            {this.renderClearButton()}
-          </div>
-        }
-        popperContainer={this.props.popperContainer}
-        popperComponent={calendar}
-        popperPlacement={this.props.popperPlacement}/>
+      <React.Fragment>
+        <div ref={(container) => { this.datePickerContainer = container; }} className="react-datepicker__input-container">
```

Comment:
Not a huge deal, but it'd be nice to start using `createRef` for our ref needs: https://reactjs.org/docs/refs-and-the-dom.html#creating-refs

Reply:
Good call, updated in this commit: https://github.com/cerner/terra-framework/pull/601/commits/dea77f089b4c8d6dd9e45945272ad2e9d3101423